### ISSUE TITLE
Audit MVP for form instructions associated with inputs

### ISSUE THREAD

**Issue by amycole501** [GSA/usagov-benefits-eligibility#998]
The MVP is ready for form instructions associated with inputs accessibility review:

Log into VDI

Access Benefits Finder MVP in a browser (Try Chrome or Edge)
https://bears-waf.app.cloud.gov/bears/life_event/death

Document issues here
https://docs.google.com/document/d/1DuIBVSXNBw8UmxvrZXEX8pb6MRklNw5vRYDpFLajux8/edit

Focus on form instructions and inputs to follow the 18F recommended critical checklist
https://docs.google.com/document/d/1YLwn3_N9XeeluZNerTojZTc71C1sDhNPDE2OrkU7GOE/edit?usp=sharing

[WCAG form guidance is here](https://www.w3.org/WAI/tutorials/forms/instructions/)
18F form testing guidance is here https://accessibility.18f.gov/forms/

Success Criteria:

[1.3.1 Info and Relationships:](https://www.w3.org/WAI/WCAG21/quickref/#qr-content-structure-separation-programmatic) Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text. (Level A)
[2.4.6 Headings and Labels:](https://www.w3.org/WAI/WCAG21/quickref/#qr-navigation-mechanisms-descriptive) Headings and labels describe topic or purpose. (Level AA)
[3.3.2 Labels or Instructions:](https://www.w3.org/WAI/WCAG21/quickref/#qr-minimize-error-cues) Labels or instructions are provided when content requires user input. (Level A)
[4.1.2 Name, Role, Value:](https://www.w3.org/WAI/WCAG21/quickref/#qr-ensure-compat-rsv) For all user interface components (including but not limited to: form elements, links and components generated by scripts), the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies. (Level A)

It is a failure if a form element isn’t programmatically associated with all instructions. This includes legends, labels, hint text and tooltips.
A common way of achieving this is using fieldset and legend tags. Fieldset is used to group a set of elements. Legend is the first child of a fieldset tag and provides context for those fields.

**Comment by amycole501** [comment]
I noted in the bug tracker that no ARIA labels were found.

This resource identifies all the WCAG success criteria for proper labelling of a form and providing instructions.
Similarly the heading levels on this page don't guide me through the form.
Also, the Month/day/year do not include elements referenced:


![Image](https://github.com/GSA/usagov-benefits-eligibility/assets/117778073/eb262dfa-f445-4592-87bb-d701b083f94f)

**Comment by amycole501** [comment]
The issues I noted above were in Storybook; when I tried accessing anything through VDI it didn't work. The system must be down (VDI, not our app). Will try again next week.
